cetmix_tower_server/tests/test_command.py

```diff
@@ -644,6 +645,49 @@ def test_user_access_rule(self):
             msg="Command name should be same",
         )
 
+    def test_user_access_rule_with_keys(self):
+        """Test user access rule"""
+
+        # create server key
+        server_key = self.Key.create(
+            {
+                "name": "server key",
+                "secret_value": "server key value",
+                "key_type": "s",
```

Comment:
I would suggest to add access level explicitly in order to simplify possible error investigation in case default access level is somehow changed, eg in another module.